src/utils/crypto.js

```diff
@@ -0,0 +1,8 @@
+const crypto = require('crypto')
+
+exports.md5 = (rawData) => {
+  if (typeof rawData !== 'string') {
+    rawData = JSON.stringify(rawData)
```

Comment:
q:    rawData must be a JSON ?

Reply:
rawData might be series of types(number, json, etc)